graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/KotlinClientApiGenTest.kt

```diff
@@ -968,5 +1000,11 @@ class KotlinClientApiGenTest {
         assertThat(actorTypeSpec.name).isEqualTo("Movies_ActorsProjection")
         val agentTypeSpec = codeGenResult.clientProjections[4].members[0] as TypeSpec
         assertThat(agentTypeSpec.name).isEqualTo("Movies_Actors_AgentProjection")
+
+        compile(codeGenResult.clientProjections)
+    }
+
+    private fun compile(dataTypes: List<FileSpec>) {
```

Comment:
What's the point of this method if it just calls one other method?

Reply:
Good catch, I had other code in there that I have since removed, so now it only does one thing. Will update.